Doc/library/pathlib.rst

```diff
@@ -1072,13 +1072,14 @@ call fails (for example because the path doesn't exist):
 
    .. versionadded:: 3.5
 
-Correspondence to tools in the os package
------------------------------------------
+Correspondence to tools in the :mod:`os` package
```

Comment:
thanks, I missed that correction from last comment, I thought only the link was missing. Fixing in next commit.